src/posthog-core.ts

```diff
@@ -714,8 +717,6 @@ export class PostHog {
             return
         }
 
-        const start_timestamp = this.persistence.remove_event_timer(event_name)
```

Comment:
I'm confused if this works at all? The companion, `set_event_timer` is never called, so this seems to always return `undefined` ?

Reply:
Hmmm that could very well be. I just noticed it didn't need to be in the parent but perhaps it doesn't work... (I actually don't even know what it does)

Reply:
Yeah, I'm not sure either.